client/src/modules/reports/generate/patientStanding/patientStanding.js

```diff
@@ -0,0 +1,72 @@
+angular.module('bhima.controllers')
+  .controller('patientStandingController', PatientStandingController);
+
+PatientStandingController.$inject = [
+  '$state', '$sce', 'NotifyService', 'BaseReportService', 'AppCache',
+  'BaseReportService', 'reportData',
+];
+
+/**
+ * @function PatientStandingController
+ *
+ * @description
+ */
+function PatientStandingController($state, $sce, Notify, BaseReportService, AppCache, SavedReports, reportData) {
+
+  const vm = this;
+  const cache = new AppCache('configure_patientStanding');
+  let reportUrl = '/reports/finance/financialPatient/';
+
+  vm.reportDetails = {};
+
+  checkCachedConfiguration();
+
+  vm.requestSaveAs = function requestSaveAs() {
+    const options = {
+      url : reportUrl,
+      report : reportData,
+      reportOptions : angular.copy(vm.reportDetails),
+    };
+
+    return SavedReports.saveAsModal(options)
+      .then(() => {
+        $state.go('reportsBase.reportsArchive', { key : options.report.report_key });
+      })
+      .catch(Notify.handleError);
+  };
+
+  // register the patient search api
+  vm.onPatientSearchApiCallback = function onPatientSearchApiCallback(api) {
```

Comment:
Do you need this function?  It doesn't look like you ever use the patient search api.